integration/integration_test.go

```diff
@@ -7371,16 +7379,32 @@ func startSSHServer(t *testing.T, caPubKeys []ssh.PublicKey, hostKey ssh.Signer)
 			_ = nConn.Close()
 		})
 
-		conn, _, reqs, err := ssh.NewServerConn(nConn, &sshCfg)
+		conn, channels, reqs, err := ssh.NewServerConn(nConn, &sshCfg)
 		require.NoError(t, err)
 		t.Cleanup(func() {
 			_ = conn.Close()
 		})
+		go ssh.DiscardRequests(reqs)
 
-		req := <-reqs
-		require.NoError(t, req.Reply(true, nil))
+		var agentForwarded bool
+		for channelReq := range channels {
+			require.Equal(t, "session", channelReq.ChannelType())
```

Comment:
`require` doesn't work in goroutine https://github.com/stretchr/testify/issues/772

Reply:
Didn't realize that thanks, updated to use `assert` package instead of `require` which calls `t.Error` instead of `t.FailNow` on a failed check